scripts/pypi_build

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I don't see anything that looks obviously wrong to me ... if the test publication works as expected, than it should be good.